src/problems/RadBeam/test_radiation_beam.cpp

```diff
@@ -38,6 +38,7 @@ template <> struct RadSystem_Traits<BeamProblem> {
 	static constexpr double radiation_constant = radiation_constant_cgs_;
 	static constexpr double Erad_floor = 0.;
 	static constexpr int beta_order = 1;
+	static constexpr bool enable_dust = false;
```

Comment:
I'd prefer that we find a somewhat more specific name for this, since dust can mean a whole lot of things. Maybe `enable_imperfect_dust_gas_coupling` or `enable_independent_dust_temp`?

Reply:
How about `enable_dust_gas_thermal_coupling`?

Reply:
Might be slightly confusing, since in fact if this is off we are assuming perfect dust-gas coupling, not disabling coupling. How about `enable_dust_gas_thermal_coupling_model` to make it clear that we are turning a model for this effect on and off.